### ISSUE TITLE
LXQT build fails due to missing header file

### ISSUE THREAD

**Issue by szg0000** [lxde/lxqt#1275]
Here is the build output. The error messages says: **fatal error: ui_selectwnd.h:File not found**

```
[ 40%] Building CXX object src/CMakeFiles/lxqt-config.dir/LXQtAppTranslationLoader.cpp.o
[ 41%] Building CXX object lxqt-config-file-associations/CMakeFiles/lxqt-config-file-associations.dir/applicationchooser.cpp.o
[ 41%] Building CXX object lxqt-config-monitor/CMakeFiles/lxqt-config-monitor.dir/managesavedsettings.cpp.o
[ 41%] Linking CXX executable lxqt-config
[ 41%] Building CXX object lxqt-config-file-associations/CMakeFiles/lxqt-config-file-associations.dir/main.cpp.o
[ 51%] Built target lxqt-config
Scanning dependencies of target lxqt-config-appearance
[ 51%] Building CXX object lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/main.cpp.o
In file included from /home/szg/lxqt/lxqt-config/lxqt-config-appearance/main.cpp:37:0:
/home/szg/lxqt/lxqt-config/lxqt-config-appearance/../liblxqt-config-cursor/selectwnd.h:26:26: fatal error: ui_selectwnd.h: Nincs ilyen fájl vagy könyvtár
 #include "ui_selectwnd.h"
                          ^
compilation terminated.
lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/build.make:211: recipe for target 'lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/main.cpp.o' failed
make[2]: *** [lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:531: recipe for target 'lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/all' failed
make[1]: *** [lxqt-config-appearance/CMakeFiles/lxqt-config-appearance.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 51%] Building CXX object lxqt-config-monitor/CMakeFiles/lxqt-config-monitor.dir/LXQtAppTranslationLoader.cpp.o
[ 51%] Building CXX object lxqt-config-file-associations/CMakeFiles/lxqt-config-file-associations.dir/LXQtAppTranslationLoader.cpp.o
[ 51%] Building CXX object lxqt-config-monitor/CMakeFiles/lxqt-config-monitor.dir/lxqt-config-monitor_autogen/moc_compilation.cpp.o
[ 52%] Building CXX object lxqt-config-file-associations/CMakeFiles/lxqt-config-file-associations.dir/lxqt-config-file-associations_autogen/moc_compilation.cpp.o
[ 52%] Linking CXX executable lxqt-config-file-associations
[ 53%] Linking CXX executable lxqt-config-monitor
[ 60%] Built target lxqt-config-file-associations
[ 69%] Built target lxqt-config-monitor
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
szg@debian:~/lxqt$
```

**Comment by fhede** [comment]
I have this issues with gentoo.
https://bugs.gentoo.org/show_bug.cgi?id=615724

**Comment by luis-pereira** [comment]
Fixed by lxde/lxqt-config@bca652a75f8a497a69b1fbc1c7eaa353f6b4eef8
